content/2022-munich/giggity.json

```diff
@@ -0,0 +1,24 @@
+{
+    "version": 2019102900,
+    "url": "https://promcon.io/2019-munich/schedule.xml",
+    "title": "PromCon EU 2019",
+        "start": "2019-11-07",
+        "end": "2018-11-08",
```

Comment:
Looks like all these days here need updating, also in the website link below?

Reply:
Thanks!